functions/src/events/index.ts

```diff
@@ -0,0 +1,46 @@
+
+import dedent from 'dedent-js'
```

Comment:
Space at the top of the file? 😱

Reply:
How is it possible?! We have a lint rule about this!!

Reply:
@antonyharfield - Seems `standard-with-typescript` was overriding my rule. I've fixed it!